src/Server/DB.fs

```diff
@@ -54,8 +58,9 @@ let private initSessionDir stime =
   |> fun info -> info.FullName
 
 let private initStudents () =
-  Path.Combine (dbdir, studentfile)
-  |> File.ReadLines
+  let dir = Path.Combine (dbdir, studentfile)
+  assertDBExistence dir
```

Comment:
Although there's no big difference in this particular case, showing a well-formed error message is a good practice.